.github/workflows/code_scan.yml

```diff
@@ -7,6 +7,26 @@ on:
     - cron: "0 18 * * 1-5"
 
 jobs:
+  Bandit:
+    runs-on: ubuntu-20.04
```

Comment:
Do we need this? The CI already runs on ubuntu 20 image. If we add this it might unnecessarily pull the image and given the proxy settings, pulling images generally fails stating "too many requests"

Reply:
it's about using the Github-hosted runner instance instead of Self-hosted one. so we don't need to worry about the image pulling burden at all.
in case of Snyk, scanning request should be passed to the internal resource but it's not possible to access it from external (GH-hosted one) without having something like VPN. so that's the reason two jobs run on different machine at this point.